shuffle-plugin/pom.xml

```diff
@@ -65,6 +65,32 @@
         </dependency>
     </dependencies>
 
+    <profiles>
+       <profile>
+            <id>with-classifier</id>
+            <dependencies>
+                <dependency>
+                    <groupId>com.nvidia</groupId>
+                    <artifactId>rapids-4-spark-sql_${scala.binary.version}</artifactId>
+                    <version>${project.version}</version>
+                    <classifier>${spark.version.classifier}</classifier>
```

Comment:
We don't need to worry about the base dependencies of this module also trying to pull in rapids-4-spark-sql without a classifier?  I expected to see a `default` vs `with-classifier` profile setup like the other poms.  This comment applies to a lot of other poms doing the same thing below.

Reply:
will make it consistent